### DISCUSSION TITLE
Welcome to ifm3d Discussions!

### DISCUSSION THREAD

**Discussion by graugans** [ifm/ifm3d#282] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GarrettPlace** [comment]
My name is Garrett Place and I am responsible for USA Product Management of the ifm 3D camera platforms.

**Comment by lola-masson** [comment]
Hello! My name is Lola Masson and I am the new Software Support Engineer for the ifm 3D cameras in robotic applications.

**Comment by GreNait** [comment]
My name is Sebastian (GreNait) and I am an application engineer based in Germany. 

Keep rockin!
-GreNait

**Comment by inbangsa** [comment]
Hello All, 

My name is Sachin (@inbangsa) , developing and supporting ifm3d  from last few years, working as Senior Software Engineer with ifm engineering pvt. ltd. Pune, India.

**Reply by graugans** [reply]
@inbangsa thanks for all your support

**Reply by inbangsa** [reply]
@graugans its all pleasure to work with you and a global team. It was a great learning experience for me during all this years.

**Comment by graugans** [comment]
Being part of the Embedded Software team at ifm my job is to take care of the 3D camera platforms. I am the software architect of the 3D camera embedded software application. I started my journey on 3D TOF cameras at ifm back in 2006. For ifm3d I consider myself as the bridge between the embedded code base and the PC side. Like @GreNait I am located in the southern part of Germany.

**Comment by RamCoremaTic** [comment]
Hi,
I am working on 3d reconstruction of an object by using two ifm O3d302 camera. I want to merge 3d point clouds taken from both camera. I have followed a given example files and got cloud pointer. I am just wondering how to visualise that in my c++

**Reply by graugans** [reply]
Hey @RamCoremaTic maybe this helps as a starting point: https://github.com/ifm/ifm3d-pcl-viewer

**Reply by RamCoremaTic** [reply]
Hi @graugans  Thanks for the reply. If I am not wrong, the above mentioned ifm3d-pcl-viewer grab point cloud directly from camera buffer and display it. But,What I want to try to do is, get the point cloud from camera and do some manipulation and then visualise it. I am just wondering is there any function in ifm3d-pcl-viewer I could use toi visualise it by sending point cloud file or its pointer as an argument. or do I need to follow and implement the pcl library functions (as it is done in the ifm3d-pcl-viewer files) to visualise it. Thank you

**Reply by lola-masson** [reply]
Hi @RamCoremaTic. We don't have any functions doing what you are looking for in the pcl-viewer, I believe you will need to adapt the code.

**Comment by scoperesolutor** [comment]
Hello does the `ifm3d` and corresponding `ifm3d-examples` apply to the O3D sensor versions too?  ifm Australia testing code out for customers. It looks like it does but i could be wrong. Thanks

**Reply by lola-masson** [reply]
Hi @scoperesolutor. The sensor can only be used with the Vision Assistant (that you can download on ifm.com).

**Reply by lola-masson** [reply]
@scoperesolutor I mis-answered your question. You actually can use ifm3d with the sensor version, to receive application data and images. The configuration of the application is done in the Vision Assistant. We do not recommend using ifm3d for configuring the application, as it would be tedious (the configuration is held in json format), but you can use ifm3d to clone already set configurations.
Hope this makes sense!

**Reply by scoperesolutor** [reply]
HI @lola-masson thanks, I had a little panic session then. It is only to receive the images and do further processing.

**Reply by lola-masson** [reply]
So sorry about that. You should be good with ifm3d then. Don't hesitate to ask if you have any further question.
